dpsim/src/PFSolver.cpp

```diff
@@ -321,6 +321,19 @@ void PFSolver::determineNodeBaseVoltages() {
             "Choose base voltage {}V of {} to convert pu-solution of {}.",
             baseVoltage_, gen->name(), node->name());
         break;
+      } else if (std::shared_ptr<CPS::SP::Ph1::Load> load =
+                     std::dynamic_pointer_cast<CPS::SP::Ph1::Load>(comp)) {
+        baseVoltage_ = load->attributeTyped<CPS::Real>("V_nom")->get();
```

Comment:
If `mNomVoltage` is intended to be of type `Attribute::Ptr` and is publicly accessible, consider directly accessing its value through the member variable instead of using a hard-coded string. For example:
`baseVoltage_ = load->mNomVoltage->get()`
This will ensure compile time-error checking. Now, if there is a typo in the string "V_nom" the error would not be caught at compile time.